tendrl/commons/utils/monitoring_utils.py

```diff
@@ -26,3 +26,28 @@ def update_dashboard(res_name, res_type, integration_id, action):
     ).save()
 
     return _job_id
+
+
+def update_graphite(res_name, res_type, integration_id, action):
+    _job_id = str(uuid.uuid4())
+    _params = {
+        "TendrlContext.integration_id": NS.tendrl_context.integration_id,
+        "TendrlContext.cluster_name": NS.tendrl_context.cluster_name,
+        "Trigger.resource_name": res_name,
+        "Trigger.resource_type": res_type,
+        "Trigger.action": action
+    }
+    _job_payload = {
+        "tags": ["tendrl/integration/monitoring"],
+        "run": "monitoring.flows.UpdateGraphite",
```

Comment:
Change flow name to 

monitoring.flows.DeleteResourceFromGraphite

https://github.com/Tendrl/monitoring-integration/pull/154/files#r142021865

Reply:
Done